docs/services/deployment/kubernetes.md

```diff
@@ -86,10 +86,10 @@ spec:
         - image: path.to/yourrepo:yourtag
           env:
             - name: PORT
-              value: 8080
+              value: 9080
```

Comment:
Not sure this one is needed, knative should inject that for you https://github.com/knative/specs/blob/main/specs/serving/runtime-contract.md#protocols-and-ports

Reply:
so, you would remove it? i didnt write this afaik

Reply:
I guess we can remove it yes.